contracts/BSCValidatorSet.sol

```diff
@@ -1054,7 +1200,11 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
     isFelony = false;
     if (slashCount >= felonyThreshold) {
       _felony(validator, index);
-      ISlashIndicator(SLASH_CONTRACT_ADDR).sendFelonyPackage(validator);
+      if (IStakeHub(STAKE_HUB_ADDR).consensusToOperator(validator) != address(0)) {
+        ISlashIndicator(SLASH_CONTRACT_ADDR).downtimeSlash(validator, slashCount);
+      } else {
+        ISlashIndicator(SLASH_CONTRACT_ADDR).sendFelonyPackage(validator);
```

Comment:
We have some risk here: 
Image we have one BC validator, and the second sunset hardfork happen and close the channel, it is possible to trigger the `sendFelonyPackage` here and revert. We should check all the place like such situation.